src/Core/Crawl/controlpos.cpp

```diff
@@ -0,0 +1,44 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "controlpos.h"
+
+namespace CRAWL {
+
+ControlPos::ControlPos() {
+
+}
+
+void ControlPos::add(ClasterItem *object) {
+
+    GroupObject::updatePosition();
```

Comment:
update position must be contains two aruments (id of item that you wnat change position, and new position )